tests/Keycloak/Listeners/CreateClientsTest.php

```diff
@@ -8,30 +8,34 @@
 use App\Domain\Integrations\Integration;
 use App\Domain\Integrations\Repositories\IntegrationRepository;
 use App\Keycloak\Client;
-use App\Keycloak\ClientCollection;
 use App\Keycloak\Config;
 use App\Keycloak\Listeners\CreateClients;
 use App\Keycloak\Realm;
 use App\Keycloak\RealmCollection;
 use App\Keycloak\Repositories\KeycloakClientRepository;
 use App\Keycloak\ScopeConfig;
 use App\Keycloak\Service\ApiClient;
+use App\Keycloak\Service\CreateClientForRealms;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
 use Ramsey\Uuid\Uuid;
 use Tests\IntegrationHelper;
-use Tests\Keycloak\KeycloakHelper;
 
 final class CreateClientsTest extends TestCase
 {
     use IntegrationHelper;
-    use KeycloakHelper;
```

Comment:
Was this not something that got changed to something more specific than `Helper`?